### PR TITLE
Fix: validate image name & help enter secret in imagePullSecrets

### PR DESCRIPTION
**Pull request by wangbow**
Signed-off-by: wb-wb665667 <wb-wb665667@alibaba-inc.com>

### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: src/api/project.ts [hunk 1]
```diff
@@ -98,3 +98,8 @@ export function getProjectConfigs(query: ProjectName) {
   const urlPath = project + `/${query.projectName}/configs`;
   return get(urlPath, {}).then((res) => res);
 }
+
+export function checkImageName(query: { image: string }) {
```
Comment (barnettZQG) [inline src/api/project.ts:102]:
```suggestion
export function checkImageName(query: { image: string, project: string }) {
```

File: src/extends/ImageInput/index.tsx [hunk 2]
```diff
@@ -23,9 +25,24 @@ class ImageInput extends React.Component<Props, State> {
...
+        imagePullSecret.push(data.sercet);
+        onChangeImagePullSecretsRef(imagePullSecret);
+      }
+    });
+  };
+
   render() {
     const { value } = this.props;
-    return <Input {...this.props} defaultValue={value} />;
+    return <Input {...this.props} defaultValue={value} onBlur={this.onCheckImageName} />;
```
Comment (barnettZQG) [inline src/extends/ImageInput/index.tsx:45]: If the image does not exist, please show the notice message

**Review by zzxwill — APPROVED** [review]

File: src/api/project.ts (hunk 1 above)
Comment (wangbow) [reply src/api/project.ts]: done

File: src/extends/ImageInput/index.tsx (hunk 2 above)
Comment (wangbow) [reply src/extends/ImageInput/index.tsx]: done

**Pull request closed without merging**